renderer/brave_wallet/brave_wallet_render_frame_observer.cc

```diff
@@ -76,6 +78,42 @@ void BraveWalletRenderFrameObserver::DidCreateScriptContext(
   }
 }
 
+void BraveWalletRenderFrameObserver::DidFinishLoad() {
```

Comment:
why are you doing this in DidFinishLoad?

Reply:
I assumed that the page load was a reasonable deadline for a third-party extension to inject their provider